ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -239,6 +239,55 @@ periodics:
         - name: config
           configMap:
             name: image-mirror-mappings
+  - agent: kubernetes
+    labels:
+      ci.openshift.io/role: image-mirroring
+      ci.openshift.io/area: codeready-toolchain
+    cron: "@midnight"
```

Comment:
Ouch, `@hourly`. I need to fix this in the template.

Reply:
done in  e646fd2